hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/ExecutionContext.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.om.ratis;
+
+import org.apache.ratis.server.protocol.TermIndex;
+
+/**
+ * context required for execution of request.
+ */
+public class ExecutionContext {
+  private long index;
```

Comment:
Why we need the `index` explicitly in this `ExecutionContext` ? Is there a case where this index cannot be determined using `TermIndex.getIndex()`

Reply:
> Why we need the `index` explicitly in this `ExecutionContext` ? Is there a case where this index cannot be determined using `TermIndex.getIndex()`

Yes,
We are going with Ozone managed Index, which will be different than termIndex. So this is kept separate. Further PR will do this behavior changes.